components/core/src/clp_s/search/antlr_common/ErrorListener.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef CLP_S_SEARCH_ANTLRCOMMON_ERRORLISTENER_HPP
+#define CLP_S_SEARCH_ANTLRCOMMON_ERRORLISTENER_HPP
+#include <antlr4-runtime.h>
```

Comment:
```suggestion

#include <cstddef>
#include <exception>
#include <string>
#include <string_view>

#include <antlr4-runtime.h>
```
Note: string_view is required if the following comment is appiled